server/server.go

```diff
@@ -145,6 +145,7 @@ func (c *Server) setupRouter(ctx context.Context, logger *logrus.Logger) (contex
 	}
 
 	r.Mount("/v1/blindedToken", c.tokenRouter())
+	r.Mount("/v1/newBlindedToken", c.markedTokenRouter())
```

Comment:
Should this be `/v2/blindedToken` instead as we are adding the cohort as a change to the existing API?